ReactTask/src/components/CardPatientProfile/CardPatientProfile.js

```diff
@@ -9,34 +9,64 @@ import {
   StylBoxDoctor,
   BoxDataPatient,
   BoxInfoPatient,
-  BtnMore,
-  StylBoxPatientsList,
+  StylBtnMore,
+  StylBoxMenuCard,
+  StylBtnChooseAction,
 } from '../index';
 
 import patient from './../../img/avatars/patients-1.png';
```

Comment:
You can delete this line, any usage

Reply:
I use this img
It is used if the user is not logged in.